src/qibolab/_core/instruments/qblox/platform.py

```diff
@@ -0,0 +1,97 @@
+from collections import defaultdict
+from typing import Optional, Union
+
+from qibolab._core.components.channels import (
+    AcquisitionChannel,
+    Channel,
+    DcChannel,
+    IqChannel,
+)
+
+__all__ = ["map_ports"]
+
+
+def _eltype(el: str):
+    return "qubits" if el[0] == "q" else "couplers"
```

Comment:
I know this is WIP and I am not sure exactly how it is used, but I would expect drivers to be qubit/coupler agnostic and only play with channels and configs, without caring exactly which qubit each channel controls, but maybe only the channel type.

Reply:
This is just a support function for writing "simpler" platforms, which I moved from the current sketch of the platform.

The idea is that using this function is fully optional. But the function itself is opinionated, favoring certain layouts (hopefully, the most frequent ones), making it simpler to describe the connections.
An example of its usage is here:
https://github.com/qiboteam/qibolab_platforms_qrc/blob/e4011ded804135d758310082616860bb959d94fd/iqm5q/platform.py#L16-L27

To be fair, the reason why I introduced this is that we're passing `qubits` and `couplers` separately in the platform, and consequently defining them in two separate sequences.
However, I refactored many times to improve the representation and implementation, and by now there is a single relic for that separation, i.e.:
https://github.com/qiboteam/qibolab/blob/1514f00ad8ccb4788d8d869b81ec183ea29416d6/src/qibolab/_core/instruments/qblox/platform.py#L93-L96

All the rest is already performed in the very same way.
So, since it is just post-processing, I should be able to concatenate `qubits` and `couplers`, and just post-process `qubits` for the acquisition.

Reply:
I have not looked at the full content of the file in detail so maybe it is indeed useful, however my main concern (and what I don't understand) is why this is under qblox. I would expect the instrument to have no knowledge about `qubits` and `couplers` at all, as the `Platform` never communicates these objects to the instrument.

If it is actually useful, it would make more sense to lift this outside the drivers and make it usable by any platform.